packages/byline-parser/index.js

```diff
@@ -34,7 +34,8 @@ export default function (blobSource, { openingPattern, closingPattern, matchPatt
         throw new Error('No capture group is defined');
       }
 
-      tmp.push(captureGroup.slice(1));
+      if (!noGroup) tmp.push(captureGroup.slice(1));
+      if (noGroup) results.push(captureGroup.slice(1));
```

Comment:
Oh, I see now. I was aware of the multiple import syntaxes, but I didn't see the need to omit the opening/closing patterns in order to ensure that `import "custom.tld/path"` isn't linked. To me, it looked like the intent was to match text like:

```golang
"os"
"github.com/foo/bar"
```

that simply doesn't have the `import (` or the `)`. It all makes sense now, though. Thanks for the clarification!